.github/workflows/code_review.yml

```diff
@@ -2,32 +2,36 @@ name: Code Review
 
 on:
   pull_request:
```

Comment:
* The `createPlayers` method now includes a lambda expression in its map function.